### ISSUE TITLE
Not stable text after animation

### ISSUE THREAD

**Issue by Gregikkps** [aagarwal1012/Animated-Text-Kit#235]
Hi, the problem is that after the end of the animation, text changes its position.

Here's a piece of code:

```
Container(
              color: Colors.amber,
              alignment: Alignment.center,
              width: 350,
              height: 250,
              child: AnimatedTextKit(
                animatedTexts: [
                  WavyAnimatedText(
                    'E x a m p l e',
                    speed: const Duration(milliseconds: 200),
                    textStyle: TextStyle(
                        fontFamily: 'UniviaProUltra',
                        fontSize: 50,
                        fontWeight: FontWeight.bold),
                  ),
                ],
                repeatForever: true,
              ),
            ),
```

![not stable text](https://user-images.githubusercontent.com/76614984/114189519-93216100-994a-11eb-87df-cce585adbd12.gif)

Thanks for every tip!

**Comment by Sameerkash** [comment]
Might be that the container is not enough for the animation? and it's pushing the text down.

**Comment by SirusCodes** [comment]
> Might be that the container is not enough for the animation? and it's pushing the text down.

I think that this is the issue.

**Comment by Gregikkps** [comment]
I have increased it to 600 and the problem is the same.

**Comment by Mohitmadhav** [comment]
> Might be that the container is not enough for the animation? and it's pushing the text down.

I think that is correct, but @Gregikkps can you try the text without any containing with just padding itself?

**Comment by Gregikkps** [comment]
@Mohitmadhav I removed all containing and is same. I noticed that with other fonts the problem sometimes disappears.

```
body: Center(
        child: AnimatedTextKit(
          animatedTexts: [
            WavyAnimatedText(
              'E X A M P L E',
              speed: const Duration(milliseconds: 200),
              textStyle: TextStyle(
                  fontFamily: 'UniviaProUltra',
                  fontSize: 50,
                  fontWeight: FontWeight.bold),
            ),
          ],
          repeatForever: true,
        ),
      ),
```


![Android Emulator - Pixel_3a_XL_API_30_5554 2021-04-20 11-51-48](https://user-images.githubusercontent.com/76614984/115380028-6edb4500-a1d2-11eb-8b86-0e8e868042b3.gif)

**Comment by SirusCodes** [comment]
Sorry for late reply.
I was able to reproduce this and also found a fix to it but also encountered another.

After fixing the position issue I found out that there are some scaling issues.

For detail Wavy uses `CustomPainter` while animating and sets a text normal `Text` widget otherwise. Now while animating text (on Canvas) size is bigger than the normal widget.

I need to confirm that if this is an issue from Flutter framework or an issue with how the package is configured.
